### PR TITLE
update invalid rules for market creation

### PR DESCRIPTION
**Pull request by bthaile**
https://github.com/augurproject/augur/issues/4515

new invalid rules for market creation

Screen Shot 2019-11-04 at 11 59 10 AM

### CODE REVIEW

**Review by phoebemirman — COMMENTED** [review]
dnt think we want all explainer blocks to have bullet points
Screen Shot 2019-11-04 at 10 08 19 AM

File: packages/augur-ui/src/modules/create-market/components/common.tsx [hunk 1]
```diff
@@ -266,7 +266,7 @@ export const ExplainerBlock = (props: ExplainerBlockProps) => (
   <div className={Styles.ExplainerBlock}>
     <h2>{props.title}</h2>
     {props.subtexts.map((subtext, index) => (
-      <p key={index}>{subtext}</p>
+      <li key={index}>{subtext}</li>
```
Comment (phoebemirman) [inline packages/augur-ui/src/modules/create-market/components/common.tsx]: shouldnt you change the styles to li as well

Comment (phoebemirman) [inline packages/augur-ui/src/modules/create-market/components/common.tsx]: prob need to address

**Review by phoebemirman — CHANGES_REQUESTED** [review]
see comments

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 💙

 This link will always point to this PR's latest changes: https://peek.run/l74he43

File: packages/augur-ui/src/modules/create-market/components/common.tsx (hunk 1 above)
Comment (bthaile) [reply packages/augur-ui/src/modules/create-market/components/common.tsx]: I did change the style,

Comment (phoebemirman) [reply packages/augur-ui/src/modules/create-market/components/common.tsx]: yea i missed that. but don't think all explainer blocks are supposed to be bulleted?

Comment (bthaile) [reply packages/augur-ui/src/modules/create-market/components/common.tsx]: updated PR

**Review by phoebemirman — APPROVED** [review]

**Pull request merged**